cache/remotecache/s3/s3.go

```diff
@@ -446,7 +446,7 @@ func (s3Client *s3Client) touch(ctx context.Context, key string) error {
 		Bucket:            &s3Client.bucket,
 		CopySource:        &copySource,
 		Key:               &key,
-		Metadata:          map[string]string{"updated_at": time.Now().String()},
+		Metadata:          map[string]string{"updated-at": time.Now().String()},
```

Comment:
I'm not very familiar with this code and how this information is used, but saw the PR; wondering; would there be code that expects the previous (`updated_at`) name? And if so, should this set _both_ values (old and new) for compatibility?


i.e, something like;

```go
updatedAt := time.Now.String()

// ....

Metadata:          map[string]string{"updated_at": updatedAt, "updated-at": updatedAt},
```

Reply:
the head `x-amz-meta-updated_at` will be dropd when it pass some gateway,  it result a change to the request, which will cause a `SignatureDoesNotMatch` error.